cpp/ql/src/semmle/code/cpp/security/Encryption.qll

```diff
@@ -49,8 +52,12 @@ string getInsecureAlgorithmRegex() {
  * Holds if `name` looks like it might be related to operations with an
  * insecure encyption algorithm.
  */
-bindingset[name] predicate isInsecureEncryption(string name) {
-  name.regexpMatch(getInsecureAlgorithmRegex())
+bindingset[name]
+predicate isInsecureEncryption(string name) {
+  name.regexpMatch(getInsecureAlgorithmRegex()) and
+  // Check for evidence that an otherwise matching name may in fact not be
+  // related to insecure encrpytion, e.g. "Triple-DES" is not "DES".
+  not name.toUpperCase().regexpMatch(".*TRIPLE.*")
```

Comment:
I think this line is equivalent to `not name.toUpperCase().matches("%TRIPLE%")`, right? It should be faster than using `regexpMatch`.

Reply:
Yes.  I'll either make this change or (quite likely) will end up deleting this bit anyway, as it's looking like we do want to flag triple-DES.

Reply:
Yep, I deleted it.